### DISCUSSION TITLE
Suspended custom domain verification

### DISCUSSION THREAD

**Discussion by Hyonam99** [vercel/community#5015] [Help] [answered]
### Summary

Please, I added a custom domain to a project, and it was successful. But after some weeks, I am getting an error message that the domain verification is suspended. Please i would appreciate any clarification or support on this.

screenshot from vercel:
![image](https://github.com/vercel/community/assets/88059700/6a73b56a-9338-4d97-bf3c-82f75e073cba)

### Example

https://www.thefxfactor.com/

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/88059700/2957a2a4-984d-4c2c-b5dd-1bd58b365ec1)

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out. It looks like your registrar has suspended the domain so it can't be used with your project. https://www.whois.com/whois/thefxfactor.com 

Please contact your registrar for more information about why it was suspended and how to recover it. 🙏
